net/hysteria/files/configs/client.yaml

```diff
@@ -0,0 +1,13 @@
+server: your.domain.net:443 
+
+auth: Se7RAuFZ8Lzg 
+
+bandwidth: 
+  up: 20 mbps
+  down: 100 mbps
+
+socks5:
+  listen: 127.0.0.1:1080 
+
+http:
+  listen: 127.0.0.1:8080
```

Comment:
dos2unix format this yaml file

Reply:
![Screenshot 2024-08-06 at 10 22 57](https://github.com/user-attachments/assets/03f39f52-78a0-4181-befd-23215b613975)

Reply:
dos2unix format this yaml file
![Screenshot 2024-08-06 at 10 22 57](https://github.com/user-attachments/assets/a3b7bd7e-c4d0-4583-a5d2-bcd52897a63c)